usvm-python/cpythonadapter/src/main/c/virtual_objects.c

```diff
@@ -278,6 +392,25 @@ allocate_raw_virtual_object(JNIEnv *env, jobject object) {
     return (PyObject *) result;
 }
 
+PyObject *
+allocate_virtual_object(JNIEnv *env, jobject object, jbyteArray mask) {
```

Comment:
The difference between "raw" virtual object and an ordinary one was that the latter had fields `ctx` and `adapter` initialized. The thing is, we can set those fields only after calling `concolicRun`, but virtual objects must be initialized before that. The object that you create here is also raw, meaning that we need to call `finish_virtual_object_initialization` on it later. So, please, return the word "raw" to avoid confusion. You may call the other function something like `allocate_raw_virtual_object_with_all_slots`.